Readme.md

```diff
@@ -1,25 +1,111 @@
 <!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/128534875/19.2.4%2B)
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E1760)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
-
-* [Default.aspx](./CS/WebSite/Default.aspx) (VB: [Default.aspx](./VB/WebSite/Default.aspx))
-* [Default.aspx.cs](./CS/WebSite/Default.aspx.cs) (VB: [Default.aspx.vb](./VB/WebSite/Default.aspx.vb))
-<!-- default file list end -->
-# ASPxGridView - How to implement select/unselect for all rows in a group row
+# Grid View for ASP.NET Web Forms- How to select/deselect all rows in a group when the control is grouped by a column
 <!-- run online -->
 **[[Run Online]](https://codecentral.devexpress.com/e1760/)**
 <!-- run online end -->
+This example demonstrates how to allow users to add/remove all rows in a group to/from selection. Note that this approach works only when the [Grid View](https://docs.devexpress.com/AspNet/5823/components/grid-view?p=netframework)'s data is grouped by a single column.
+
+![Select/Deselect a Group](select-deselect.gif)
+
+## Overview
+
+Follow the steps below to allow users to to select/deselect all rows in a group:
```

Comment:
I remove slash between operations wherever it is possible because / can be read as follows:

1. AND
2. OR
3. AND/OR

I understand there are scenarios when the alternative would have been way worse (see above). However, this is not the case, therefore, if we can add certainty here, we definitely should:

```suggestion
Follow the steps below to allow users to to select or deselect all rows in a group:
```